RELEASES.md

```diff
@@ -1,3 +1,152 @@
+Version 1.45.0 (2020-07-16)
+==========================
+
+Language
+--------
+- [Float to int conversion using `as` has been defined as a saturating conversion.][71269]
+  This was previously undefined behaviour.
+- [`mem::Discriminant<T>` now uses `T`'s discriminant type instead of always
+  using `u64`.][70705]
+- [Procedural macros can now be used in expression, pattern, and  statement
```

Comment:
Function-like procedural macros.
Attribute macros still cannot be used in those positions.